imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -71,6 +71,7 @@ class CaseExplorer extends Component {
       })
     }
   }
+
```

Comment:
The empty lines should not be committed (there are no other changes in this file)